src/en/wip-systems.md

```diff
@@ -45,19 +45,33 @@ like Landscape and Nagios.
 
 ## Creating environments in a Juju System
 
+> Somewhere it should be noted that an initial juju environment must be
+> bootstrapped before 'system create-environment' can be called.
+> Later in the discussion of `juju system destroy` it becomes clear this
+> original, bootstrapped environment is the `system` and it must be
+> bootstrapped before proceeding.
+
 The `juju system create-environment` command will create a new empty
 environment that has the current user as the owner. These secondary
 environments are called 'hosted environments', as they are hosted within a
 Juju System. When creating an environment, the user will need to specify cloud
```

Comment:
Yes, the part about creating credentials needs a lot more explanation, with a lot more examples